subprojects/test-runner/device-provider/impl/src/main/kotlin/com/avito/android/runner/devices/internal/EmulatorsLogsReporter.kt

```diff
@@ -1,8 +1,13 @@
 package com.avito.android.runner.devices.internal
 
 import com.avito.runner.service.worker.device.Serial
+import java.io.File
 
 internal interface EmulatorsLogsReporter {
+
     fun reportEmulatorLogs(emulatorName: Serial, log: String)
+
     fun redirectLogcat(emulatorName: Serial, device: Device)
+
+    fun getLogFile(podIp: String): File
```

Comment:
For future changes EmulatorsLogsReporter:
The EmulatorsLogsReporter is a strange thing. It knows too much and I don't understand its' responsibility

Maybe add TODO that will help refactor it

Reply:
i agree